packages/babel-plugin/__tests__/transform-import-export-test.js

```diff
@@ -0,0 +1,527 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+jest.autoMockOff();
+
+const { transformSync } = require('@babel/core');
+const stylexPlugin = require('../src/index');
+
+function transform(source, opts = {}) {
+  const { code, metadata } = transformSync(source, {
+    filename: opts.filename || '/stylex/packages/vars.stylex.js',
```

Comment:
This line surfaces a difference between the `importSources` tests and all the others. If you remove the fallback filename, only the `importSources` tests fail:

```
  ● @stylexjs/babel-plugin › [transform] stylex imports › importSources (string)

    /stylex/packages/fixture.js: No filename found for generating theme name.
```

And if you use the same fallback filename for every file, the `importSources` tests pass but new CSS is added to the metadata.

Either we have a bug in the compiler, or there's some esoteric detail I don't understand.